### PR TITLE
Introduction to the Scrum Framework

### PR DESCRIPTION
**Pull request by alyson647**
This pull requests contains the first draft of a new file for an introduction to the scrum framework. This topic would be located in the Software Engineering category. This topic provides an introduction to scrum, explaining what it is, the values, artifacts, events, and roles.

### CODE REVIEW

**Review by NonLan — COMMENTED** [review]
Looks good Alyson! The article is clear and very informative. I liked that you discussed its importance and linked additional resources, I'm sure these will be helpful to future students of the course.

In terms of additions, I think you should include a table of contents at the top of the article, mainly because it seems like most other pages seem to be doing that. As well, it could help with overall organization and clarity. Otherwise, content and structure looks good!

File: Topics/Software_Engineering/Scrum.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+- Daily standup:
+    - The standup is a regularly scheduled meeting in which members of the team will update members on their progress and mention blockers they are facing with their work.
+- Sprint review:
+    - This occurs at the end of the sprint, where the team meets to demo the end product and showcase the completed sprint backlog items.
+- Sprint retrospective: 
+    - Also occurring at the end of the sprint, the retro is where the team discusses the aspects of the sprint that worked and parts that could use improvement.
+    - This builds in feedback and continual improvement of processes in the scrum framework.
+
+### Why is scrum important?
+Teams use the scrum framework since it provides an efficient and adaptable way to organize and manage teams and products. It is team-centric and self-managed and encourages creativity with the flexibility to assign work based on work styles. The framework has concrete roles, events, artifacts, and values. These aspects of scrum are incorporated into professional workplaces and can be used in the 301 to finish the project in the short amount of time given.
```
Comment (NonLan) [inline Topics/Software_Engineering/Scrum.md:63]: Minor grammar thing, I would replace "the 301" with "CSC301"/"this course".

Comment (alyson647) [reply Topics/Software_Engineering/Scrum.md:63]: Thanks for that catch!

**Review by thakshaCS — COMMENTED** [review]
Hey Alyson, you use the term scrum team in what are sprints. And I was a bit confused on what a scrum team was till i saw you next section "members of a scrum team". I think you should move the section "members of a scrum team" before the section "what are sprints"

**Comment by thakshaCS** [comment]
Overall, Alyson great job! It is very well written and easy to ready. I also think its an interesting and valuable topic that you covered.

**Comment by alyson647** [comment]
> Hey Alyson, you use the term scrum team in what are sprints. And I was a bit confused on what a scrum team was till i saw you next section "members of a scrum team". I think you should move the section "members of a scrum team" before the section "what are sprints"

Thanks for the feedback! I didn't realize while I was writing or editing that I referred to the "scrum team" before I got to that section. I'll make sure to change the structure of the file so it doesn't cause any confusion.

**Comment by alyson647** [comment]
> Looks good Alyson! The article is clear and very informative. I liked that you discussed its importance and linked additional resources, I'm sure these will be helpful to future students of the course.
> 
> In terms of additions, I think you should include a table of contents at the top of the article, mainly because it seems like most other pages seem to be doing that. As well, it could help with overall organization and clarity. Otherwise, content and structure looks good!

Yes that's a good idea! It'll definitely help with organizing the page and make it easier to go to certain sections of the page.

**Pull request merged**